src/components/widget/ActionButton.js

```diff
@@ -57,18 +62,26 @@ class ActionButton extends Component {
     }
 
     handleDropdownFocus = () => {
-        const { dispatch, windowType, fields, dataId, dropdownOpenCallback} = this.props;
+        const {dropdownOpenCallback} = this.props;
 
+        this.fetchStatusList();
+        dropdownOpenCallback();
+        this.statusDropdown.classList.add('dropdown-status-open');
+    }
+
+    fetchStatusList(){
+        const { dispatch, windowType, fields, dataId} = this.props;
         dispatch(dropdownRequest(windowType, fields[1].field, dataId, null, null, "window")).then((res) => {
             this.setState({list: res.data});
         });
-        dropdownOpenCallback();
-        this.statusDropdown.classList.add('dropdown-status-open');
     }
 
     handleChangeStatus = (status) => {
-        this.props.onChange(status);
```

Comment:
If you are changing some file and it is out of our code schema, please fix it too beside.

``` const {onChange} =  ... ```